test/config-luatest/upgrade_test.lua

```diff
@@ -71,14 +71,13 @@ end)
 g.test_upgrade = function()
     g.server:exec(function()
         t.assert_equals(box.space._schema:get("version"), {'version', 2, 11, 0})
-        local messages = {}
-        for _, alert in pairs(require("config"):info().alerts) do
-            table.insert(messages, alert.message)
-        end
-        local exp = 'box.schema.user.grant("guest", "read", "space", ' ..
-            '"_space") has failed because either the object has not been ' ..
-            'created yet, a database schema upgrade has not been performed, ' ..
-            'or the privilege write has failed (separate alert reported)'
```

Comment:
Shouldn't we have some alert regarding this situation? At least some common, like 'Processing of the "credentials" section from config is postponed till database schema upgrade'.

Reply:
There used to be a verbose message about it. I made it a warning and added a test.

Reply:
Nice! If it is not a big deal, please, also drop this warning when the upgrade occurs and extend the test with a check that the warning actually disappears.

It seems that we can do it here:

```lua
if not schema_is_upgraded then
    config._aboard:set({<...>})
    schema_is_upgraded_cond:wait()
    assert(schema_is_upgraded)
    -- <..here..>
end
```

Reply:
Haven't noticed the fact the warnings had to be dropped. Thank you!